www/src/pages/index.js

```diff
@@ -19,7 +19,7 @@ function HomepageHeader() {
         <h1 className="hero__title">{siteConfig.title}</h1>
         <p className="hero__subtitle">{siteConfig.tagline}</p>
         */}
-        <h3> #ThisMonthInSWA - Nov 2022! </h3>
+        <h3> #ThisMonthInSWA - Jan 2023! </h3>
         <Link to="blog/roundup-2022-nov">
```

Comment:
Will update this with separate follow up PR once this is merges to avoid the broken link with docusaurus.